framework/src/org/checkerframework/common/wholeprograminference/WholeProgramInference.java

```diff
@@ -55,7 +56,33 @@ void updateInferredMethodParameterTypes(
             ExecutableElement methodElt, AnnotatedTypeFactory atf);
 
     /**
-     * Updates the type of {@code parameter} base on an assignment of
+     * Updates the parameter types of the method {@code methodTree} based on the
+     * parameter types of the overridden method {@code overriddenMethod}.
+     * @param methodTree the tree of the method that contains the parameter.
+     * @param methodElt the element of the method.
+     * @param overriddenMethod the overridden method.
```

Comment:
Recommend: "the AnnotatedExecutableType of the overridden method". Same for the Javadoc of updateInferredMethodReceiverType.